services/data/db_utils.py

```diff
@@ -60,3 +51,20 @@ def get_exposed_task_id(task_id, task_name):
     if task_name is not None:
         return task_name
     return task_id
+
+
+def _get_latest_attempt_id_for_tasks(artifacts):
```

Comment:
the underscore prefix in this repo was meant to signify that the function shouldn't be used outside of the class or module